string group
float64 speed

---------------
bool success
